compiler/rustc_parse/src/parser/mod.rs

```diff
@@ -998,7 +1013,13 @@ impl<'a> Parser<'a> {
     }
 
     /// Advance the parser by one token using provided token as the next one.
-    fn bump_with(&mut self, (next_token, next_spacing): (Token, Spacing)) {
+    fn uninlined_bump_with(&mut self, next: (Token, Spacing)) {
```

Comment:
```suggestion
    fn bump_with(&mut self, next: (Token, Spacing)) {
```
Did you forget to push the changes?
The `uninlined_*` methods are still here.

Reply:
Sorry, I overlooked your request to remove the `uninlined_` prefix.

FWIW, the way I did it originally is how it's been done in other places within the compiler, and how it's recommended to do it in [the perf book](https://nnethercote.github.io/perf-book/inlining.html#harder-cases).